e2e/dr_test.go

```diff
@@ -21,6 +21,10 @@ func TestDR(dt *testing.T) {
 		t.Fatal("No tests found in the configuration file")
 	}
 
+	if err := util.ValidateClustersInDRPolicy(Ctx.env, Ctx.config, Ctx.log); err != nil {
+		t.Fatalf("Failed to validate clusters with DRPolicy: %s", err)
+	}
```

Comment:
We cannot add here this kind of specific validation. Soon we will have 5 different validations and we need the same validation in ramenctl. What we need is:

- Move util/valdiation.go to new validate package.
- Add ValidateConfig(*types.Env, *types.Config, *zap.SugaredLogger) error
- Call it here
- Inside ValidateConfig you can call validateClustersInDRPolic (which can be private now)

In ramenctl we wil have the same call here:
https://github.com/RamenDR/ramenctl/blob/2b42fa2c7b2dd7f9ef33fdb00a622ddf25c28d4a/pkg/test/run.go#L19

We also need the same call in valdiation_test.go.

Reply:
Ack, thanks.  Shall i take this up in a follow up PR?